scripts/staking.py

```diff
@@ -90,14 +101,36 @@
                 print(f"Service {args.service_id} is not staked.")
                 sys.exit(0)
 
+            if is_service_evicted(
+                ledger_api, args.service_id, args.staking_contract_address
+            ):
+                print("WARNING: Your service has been evicted from the staking program due to inactivity.")
+                input("Press Enter to continue...")
```

Comment:
Should we suggest to unstake here?

Reply:
This is in the unstaking flow. https://github.com/valory-xyz/trader-quickstart/pull/196/files#diff-ff2b1400d3292ccf632584d149b6808f071cf2d8dda5e0cad196034d88206634R96

Reply:
Oh, correct. Then should we show this message and suggest to unstake if the user re-runs their previously staked service and it has now been evicted?

Reply:
We could add this warning on line 95, before any "if". But we have been discussing with @mariapiamo that the best placement was here.

Reply:
Maybe here https://github.com/valory-xyz/trader-quickstart/pull/196/files#diff-ff2b1400d3292ccf632584d149b6808f071cf2d8dda5e0cad196034d88206634R171  we can can have  the following:
```
           if is_service_evicted(
                          ledger_api, args.service_id, args.staking_contract_address
                      ):
                print("Your service has been evicted from the staking program due to inactivity. Unstaking...")
                unstake_txs = get_unstake_txs(
                    ledger_api, args.service_id, args.staking_contract_address
                )
                for tx in unstake_txs:
                    send_tx_and_wait_for_receipt(ledger_api, owner_crypto, tx)
                print("Unstaked successfully.")
                sys.exit(0)
```

Reply:
Added :+1: